tasks/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "robocorp-tasks"
-version = "3.0.0"
+version = "3.0.1"
```

Comment:
Even if this looks like a breaking change now (removing the function from the public API), we agreed to mark it as a patch fix since it's affecting a file that was never meant for importing outside of our internal use and the function call won't have any effect anyway since it's already executed by the time of the import.